### ISSUE TITLE
Single Cell view: OpenCOR crashes when trying to change the properties of a graph

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#660]
Create a graph and double click on it to see its properties (i.e. its ```Model```, ```X``` and ```Y``` properties). From there, click on the value of its properties and, while the mouse button is pressed down, move to another property value. If you keep doing this then OpenCOR will eventually crash.

The problem seems to be related to the closing of the editor that is used to edit the property value:

```
7   libCore.dylib                 	0x00000001105f4676 OpenCOR::Core::PropertyEditorWidget::editorClosed() + 342 (propertyeditorwidget.cpp:1953)
```

**Comment by agarny** [comment]
FWIW, the problem doesn't exist with OpenCOR 0.4.1 (released on 20 May 2015) while it does with the latest published snapshot (2015-06-09). [Qt 5.4.2](https://blog.qt.io/blog/2015/06/02/qt-5-4-2-released/) was released on 2 June 2015 and I wonder whether it might not be the culprit since .

**Comment by agarny** [comment]
While working on this issue, I came across some code that I thought wasn't relevant anymore and therefore cleaned it up (see 54b29cac04dfb1e898827296748f7187ae96fd39) and... surprise, surprise, it fixes this issue.

A quick glance at the code (that was removed) shows that it had potential for a generating a problem like the one reported in this issue. Still, it's interesting to see that it was working 'fine' before Qt 5.4.2...

Anyway, this issue is now 'fixed', so closing.
